src/frameName.h

```diff
@@ -76,8 +76,10 @@ class FrameName {
     ThreadMap& _thread_names;
     locale_t _saved_locale;
     bool _includemm;
+    const int access_flags_size = 12;
```

Comment:
Any reason we need the const 12 *here* and in the array itself?

In C (and probably C++) you could do:
1. `const std::pair<int, std::string> access_flags []` (without a length)
2. To get the size you could do `sizeof(access_flags) / sizeof(access_flags[0])`. The project already has a macro `ARRAY_SIZE`.

Reply:
Point 1 - results with: `src/frameName.h:94:5: error: too many initializers for 'const std::pair<int, std::basic_string<char> > [0]'`.  Maybe we can just use `const std::pair<int, std::string> access_flags [12]` and then getting size by code you provided in Point 2.

Reply:
Yes, let's